### DISCUSSION TITLE
SFF Corrector Pages

### DISCUSSION THREAD

**Discussion by ashpags** [lightkurve/lightkurve#1121] [Q&A] [answered]
Hello! First, thank y'all for all of the helpful tutorials and documentation for Lightkurve -- as someone who is using both Lightkurve and Kepler data for the very first time, they have been invaluable! My undergrad research student and I are working with some K2 data and trying to replicate the SFF correction done by Vanderburg & Johnson 2014 to remove the photometric variations caused by the spacecraft drift in some short-cadence data. (We can't find anywhere this is already done by a pipeline for short-cadence data, just long-cadence. But if it exists, we'd love to know about it!) When we run Lightkurve's built-in SFF corrector, those variations are still there, so we were going to try to replicate the V&J steps by hand. There are two "further detail" pages linked in the [SFF Corrector tutorial](https://docs.lightkurve.org/tutorials/2-creating-light-curves/2-3-k2-sffcorrector.html), but neither page exists anymore. They are [https://docs.lightkurve.org/tutorials/04-replicate-vanderburg-2014-k2sff.html](https://docs.lightkurve.org/tutorials/04-replicate-vanderburg-2014-k2sff.html) and [https://docs.lightkurve.org/tutorials/04-replicate-vanderburg-2014-lightkurve.html](https://docs.lightkurve.org/tutorials/04-replicate-vanderburg-2014-lightkurve.html). I found archived versions on the Wayback Machine, but I just wanted to check and see if there's a reason those pages don't exist anymore, if there are known errors, or if there's some other reason we shouldn't use them. Thank you in advance!

**Comment by orionlee** [comment, marked as answer]
Tutorial 04-replicate-vanderburg-2014-k2sff.html was removed as part of the lightkurve v2 documentation revamp in PR #926 . So maybe @barentsen could shed some light?

**Reply by barentsen** [reply]
@ashpags Good question!  The Jupyter notebook you mention was part of the Lightkurve v1 docs.  You can view an archived version here:

https://github.com/lightkurve/lightkurve/blob/v1.11.3/docs/source/tutorials/04-replicate-vanderburg-2014-k2sff.ipynb

This notebook did not make it into the Lightkurve v2 docs yet because it does not directly demonstrate a Lightkurve feature.  It does of course provide helpful background information to help understand the inner workings of the `SFFCorrector` class, so I'd be open to see it re-instated.  There is nothing wrong with it, though beware that it does not provide a *complete* explanation of the internals of `SFFCorrector` as it is implemented right now.

Note: the SFF algorithm is more difficult to use on short-cadence data (for reasons I do not remember right now).  I'd be very interested to hear about any and all experiences in using it on K2 short cadence data.  Alternatively, the `PLDCorrector` may work well out of the box for such data.

Does this help?

**Reply by ashpags** [reply]
Yes, that helps! Thank you! And yes, we're having issues with the SFF algorithm on short-cadence data, but have some more playing around to do first before we declare it a lost cause. We'd just really like to be able to use that short-cadence data but without removing the jitter from the spacecraft motion, it's not super useful. We'll figure something out though! Thank you again!
